strawberry/field_extensions/input_mutation.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import annotations
+
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Dict,
+    TypeVar,
+)
+
+import strawberry
+from strawberry.annotation import StrawberryAnnotation
+from strawberry.arguments import StrawberryArgument
+from strawberry.extensions.field_extension import (
+    AsyncExtensionResolver,
+    FieldExtension,
+    SyncExtensionResolver,
+)
+from strawberry.field import StrawberryField
+from strawberry.utils.str_converters import to_camel_case
+
+if TYPE_CHECKING:
+    from strawberry.types.info import Info
+
+_T = TypeVar("_T")
+
+
+class InputMutationExtension(FieldExtension):
+    def apply(self, field: StrawberryField) -> None:
+        resolver = field.base_resolver
+        assert resolver
+
+        name = field.graphql_name or to_camel_case(resolver.name)
+        name_captalized = name[0].upper() + name[1:]
```

Comment:
let's use `capitalize_first` here 😊

```suggestion
        name_capitalized = name[0].upper() + name[1:]
```